script/deploy/_targetState.json

```
@@ -40,6 +39,7 @@
         "RelayerCelerIM": "2.0.0",
         "CelerIMFacetMutable": "2.0.0",
         "GasZipFacet": "2.0.2",
```

Biome (1.9.4)

[error] The key GasZipFacet was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)